### DISCUSSION TITLE
How to put 0 in front of time?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1915] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hello, I'm using the calendar module with datetime and I would like to have times displayed with zero in front of time that are less than 10.

![image](https://user-images.githubusercontent.com/5399265/111138486-48e1d580-8580-11eb-897b-b3c7744581d6.png)

Obviously, the same should happen to the time in the string that appears after the users pick date and time from calendar.

Is it possible to do that?

I have found [this](https://github.com/fomantic/Fomantic-UI/discussions/1204), but it seems not to be usefull in my case.

**Comment by lubber-de** [comment, marked as answer]
As mentioned in the linked post you can also modify the time formatter according to your needs, to

```javascript
    time: function (date, settings, forCalendar) {
      if (!date) {
        return '';
      }
      var hour = date.getHours();
      var minute = date.getMinutes();
      var ampm = '';
      if (settings.ampm) {
        ampm = ' ' + (hour < 12 ? settings.text.am : settings.text.pm);
        hour = hour === 0 ? 12 : hour > 12 ? hour - 12 : hour;
      }
      return (hour < 10 ? '0':'') + hour + ':' + (minute < 10 ? '0' : '') + minute + ampm;
    },
```
See https://jsfiddle.net/lubber/jpLru0ev/6/
![image](https://user-images.githubusercontent.com/18379884/111143289-e68bd380-8585-11eb-8283-cbfa64c48274.png)

**Comment by smartm0use** [comment]
Got it, thank you!
